even-more-fish-plugin/src/main/java/com/oheers/fish/EvenMoreFish.java

```diff
@@ -552,20 +552,15 @@ public void reload(@Nullable CommandSender sender) {
 
     // Checks for updates, surprisingly
     private boolean checkUpdate() {
-
-        String[] spigotVersion = new UpdateChecker(this, 91310).getVersion().split("\\.");
-        String[] serverVersion = getDescription().getVersion().split("\\.");
-
-        for (int i = 0; i < serverVersion.length; i++) {
-            if (i < spigotVersion.length) {
-                if (Integer.parseInt(spigotVersion[i]) > Integer.parseInt(serverVersion[i])) {
-                    return true;
-                }
-            } else {
-                return false;
-            }
+        // Removes the dots from the version numbers and turns them into ints.
+        // If spigotVersion's int is bigger than serverVersion's int, there is an update available.
+        try {
+            int spigotVersion = Integer.parseInt(new UpdateChecker(this, 91310).getVersion().replace(".", ""));
```

Comment:
There might be an issue here, maybe we use an existing library to handle version comparisons.

Let me explain:
Somewhere along the way we will have a version 2.1.23, and we introduce a hotfix or a patch to the page, 2.1.23.1, and then later on an update 2.1.24.

This code will register 2.1.23.1 as the latest version and not 2.1.24